scripts/common_test.py

```diff
@@ -375,3 +384,137 @@ def mock_getpass(prompt):
             'access token at https://github.com/settings/tokens and re-run '
             'the script'):
             common.get_personal_access_token()
+
+    def test_closed_blocking_bugs_milestone_results_in_exception(self):
+        mock_repo = github.Repository.Repository(
+            requester='', headers='', attributes={}, completed='')
+        # pylint: disable=unused-argument
+        def mock_get_milestone(unused_self, number):
+            return github.Milestone.Milestone(
+                requester='', headers='',
+                attributes={'state': 'closed'}, completed='')
+        # pylint: enable=unused-argument
```

Comment:
Why use `# pylint: disable=unused-argument` here and not rename `number` to `unused_number`? Ditto below.

Reply:
`number` is passed as a named argument, so replacing it with `unused_number` raises an error that the function got an unexpected argument `number`